website/docs/r/org_user.html.markdown

```diff
@@ -0,0 +1,165 @@
+
+~> **Note:** Only `System Administrator` or `Org Administrator` users can create users.
+
+## Example Usage
+
+```hcl
+resource "vcd_org_user" "my-org-admin" {
+  org = "my-org"
+
+  name          = "my-org-admin"
+  description   = "a new org admin"
+  role          = "Organization Administrator"
+  provider_type = "INTEGRATED"
+  password      = "change-me"
+  is_enabled    = true
+}
+
+resource "vcd_org_user" "test_user_vapp_author" {
+  org = "datacloud"
+  
+  name              = "test_user_vapp_author"
+  password_file     = "org_user_pwd.txt"
+  full_name         = "test user vapp author"
+  description       = "Org user test_user_vapp_author"
+  role              = "vApp Author"
+  is_enabled        = true
+  take_ownership    = true
+  provider_type     = "INTEGRATED"
+  stored_vm_quota   = 20
+  deployed_vm_quota = 20
+  instant_messaging = "@test_user_vapp_author"
+  email_address     = "test_user_vapp_author@test.company.org"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization to which the VDC belongs. Optional if defined at provider level.
+* `name` - (Required) A unique name for the user.
+* `password` - (Optional, but required if `password_file` was not given) The user password. This value is never returned 
+  on read. It is inspected on create and modify. On modify, the absence of this element indicates that the password 
+  should not be changed.
+* `password_file` (Optional, but required if `password` was not given). A text file containing the password. 
+  Using this property instead of `password` has the advantage that the sensitive data is not saved into Terraform state 
+  file.
+* `provider_type` - (Optional) Identity provider type for this this user. One of: `INTEGRATED`, `SAML`, `OAUTH`. The default
+   is `INTEGRATED`.
+* `role` - (Required) The role of the user. Role names can be retrieved from the organization. Both built-in roles and
+  custom built can be used. The roles normally available are:
+    * `Organization Administrator`
+    * `Catalog Author`
+    * `vApp Author`
+    * `vApp User`
+    * `Console Access Only`
+    * `Defer to Identity Provider`
+* `full_name` - (Optional) The full name of the user.
+* `description` - (Optional) An optional description of the user.
+* `telephone` - (Optional) The Org User telephone number.
+* `email_address` - (Optional) The Org User email address. Needs to be a properly formatted email address.
+* `instant_messaging` - (Optional) The Org User instant messaging.
+* `is_enabled` - (Optional) True if the user is enabled and can log in. The default is `false`.
+* `is_group_role` - (Optional) True if this user has a group role.. The default is `false`.
+* `is_locked` - (Optional) True if the user account has been locked due to too many invalid login attempts. A locked 
+  user account can be re-enabled by updating the user with this flag set to false. Only the system can set the value to 
```

Comment:
> re-enabled by updating the user with this flag set to false

Had a comment above somewhere that this phrasing is confusing.

Reply:
waiting for suggestions

Reply:
Just posted one above.

Reply:
applied